scripts/west_commands/runners/nrfjprog.py

```diff
@@ -119,18 +151,43 @@ def do_run(self, command, **kwargs):
 
         self.ensure_snr()
 
-        commands = []
         board_snr = self.snr.lstrip("0")
 
         if not os.path.isfile(self.hex_):
             raise ValueError('Cannot flash; hex file ({}) does not exist. '.
                              format(self.hex_) +
                              'Try enabling CONFIG_BUILD_OUTPUT_HEX.')
 
-        program_cmd = ['nrfjprog', '--program', self.hex_, '-f', self.family,
-                    '--snr', board_snr] + self.tool_opt
+        if self.family != "NRF53":
+            self.do_flash(self.hex_, self.tool_opt, board_snr)
+        else:
+            # self.family == "NRF53"
+
+            def flash_app(hex_file, reset=True):
+                self.do_flash(hex_file, self.tool_opt + ['--coprocessor', 'CP_APPLICATION'], board_snr, reset)
```

Comment:
```suggestion
                self.do_flash(hex_file, self.tool_opt + ['--coprocessor', 'CP_APPLICATION'], board_snr, reset=reset)
```

Just a personal preference to use keyword arguments explicitly if you don't mind.